### PR TITLE
invalid DR lines in EMBL files from RepBase

### PR DESCRIPTION
**Pull request by OscarMaestre**
Includes an example of a wrong file (DR invalid structure) in Tests/RepBase23.02.embl

This pull request addresses issue #1579 

I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

I am happy be thanked by name in the ``NEWS.rst`` and ``CONTRIB.rst`` files,
and have added myself to those files as part of this pull request. (*This
acknowledgement is optional. Note we list the names sorted alphabetically.*)

### CODE REVIEW

**Comment by OscarMaestre** [comment]
Ops, sorry, I forgot to run flake8. Reject that pull request, I will solve the problems

File: Tests/test_EMBL_unittest.py [hunk 1]
```diff
@@ -51,6 +51,13 @@ def test_embl_no_coords(self):
             self.assertEqual(rec.seq[:10], "MVLSEGEWQL")
             self.assertEqual(rec.seq[-10:], "AKYKELGYQG")
 
+    def test_embl_wrong_dr_line(self):
+        """Test files with wrong DR lines"""
+        with warnings.catch_warnings(record=True) as w:
+            warnings.simplefilter("always")
+            fasta_seq = SeqIO.read('EMBL/RepBase23.02.embl', 'embl')
+            self.assertTrue(w, "Malformed DR line in EMBL file.")
```
Comment (peterjc) [inline Tests/test_EMBL_unittest.py]: I don't think this should be ``assertTrue``, rather something like (untested):

```python
with warnings.catch_warnings(record=True) as w:
    warnings.simplefilter("always", BiopythonParserWarning)
    record = SeqIO.read('EMBL/RepBase23.02.embl', 'embl')
    self.assertEqual(len(w), 1)
    self.assertEqual(str(w[0]), "Malformed DR line in EMBL file."))
```

**Comment by peterjc** [comment]
I've updated the tests, not quite as I was suggesting but more like the previous test already in the file.

**Comment by peterjc** [comment]
Did you want to be thanked by name?

**Pull request merged**